lib/mongoid/scroll/cursor.rb

```diff
@@ -39,6 +39,14 @@ def to_s
 
       private
 
+      def compare_direction
```

Comment:
Moved to a new method to avoid triggering a Rubocop [PerceivedComplexity](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Metrics/PerceivedComplexity) offense.

Reply:
You should just run `rubocop -a ; rubocop --auto-gen-config`, but I'm also 100% for just disabling the complexity cops altogether if needed.

Reply:
I'm fine moving it to `compare_direction`, it was just to explain why I did this change which might not appear as required.